com.unity.netcode.gameobjects/Runtime/Messaging/Messages/NetworkVariableDeltaMessage.cs

```diff
@@ -54,6 +54,14 @@ public void Serialize(FastBufferWriter writer)
                     networkVariable.CanClientRead(TargetClientId) &&
                     (NetworkBehaviour.NetworkManager.IsServer || networkVariable.CanClientWrite(NetworkBehaviour.NetworkManager.LocalClientId));
 
+                // Prevent the server from writing to the client that owns a given NetworkVariable
+                // Allowing the write would send an old value to the client and cause jitter
+                if (networkVariable.WritePerm == NetworkVariableWritePermission.Owner &&
+                    networkVariable.OwnerClientId() == TargetClientId)
+                {
+                    shouldWrite = false;
+                }
```

Comment:
I know this won't sound like the best idea but packing this condition somewhere inside the `var shouldWrite = ...` statement above strategically would give us just a little performance back due to lazy evaluation nature of conditionals in C#. Just throwing this out there as an optional suggestion, I don't hold a strong opinion here :P

Reply:
I considered it and it felt like readability would trump micro-optimization. It's a separate concern of overriding a send, so it wanted it clearly separated.

Also a case where a proper compiler and language would get optimized out above what a human could do, anyway, but this is C#, so I don't really know.

Reply:
You know...I think there are some tests where we had to increase the approximation threshold in some areas to like 0.5 (or something crazy like that) due to some weird "spikes" in like the client-side delta (previous vs current).  It would be interesting to say decrease TransformInterpolationTests.maxRoundingError from 0.01 to 0.001 (or lower) and see if we no longer get those random "delta spikes"... ?

Reply:
sure, just wanted to mention, not a big deal :P